docker-compose-pool-combo-sgx.yaml

```diff
@@ -0,0 +1,86 @@
+# Copyright 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+version: '3.5'
+
+volumes:
+  # Named volume for WPE to share MRENCLAVE with KME. MRENCLAVE value can also be passed
+  # as a command line parameter (using the option --wpe_mr_enclave) to KME enclave manger.
+  # Then the MRENCLAVE value stored in file "wpe_mr_enclave.txt" in shared volume will be
+  # overridden by the value passed in command line.
+  pool-1:
+  pool-2:
+
+services:
+
+  avalon-kme-1:
+    container_name: avalon-kme-1
+    image: avalon-enclave-manager-kme-sgx-dev
+    build:
+      args:
+        - SGX_MODE=HW
+    volumes:
+      # Map aesm service on host to docker, required only in HW mode.
+      - /var/run/aesmd/aesm.socket:/var/run/aesmd/aesm.socket
+    devices:
+      - "/dev/isgx:/dev/isgx"
+
+  avalon-enclave-manager:
```

Comment:
Can we name avlon-enclave-manager as  avalon-wpe-1-1 ? Also please add a comment wpe-1-1 means 1st WPE in pool-1 that's associated with KME-1.
We have total 2 pools and 5 enclave managers in total in this compose file.
avalon-kme-1, avalon-wpe-1-1 ,avalon-wpe-2-1 (pool 1)
avalon-kme-2, avalon-wpe-1-2  (pool 2)

Reply:
Updated with comments above service name to indicate pool layout.